.github/workflows/release.yaml

```diff
@@ -42,13 +41,30 @@ jobs:
     name: Build XCFramework
     runs-on: macos-latest
     needs: [run_tests]
-    env:   
+    env:
       umbrella_xcframework_path: "umbrella-framework/build/XCFrameworks/release/JsonLogicKMP.xcframework"
+      config_file_name: "JsonLogicConfig.kt"
+      config_file_path: "core/src/iosMain/kotlin/"
     outputs:
       sha256: ${{ steps.calculate_checksum.outputs.sha256 }}
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
+        with:
+            fetch-depth: 0
+
+      - name: Get next tag
```

Comment:
Now we have two bumps here and in line `104`. How about using only one? I think it will be more readable.